<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>1200</width><height>720</height></rect>
  </property>
  <property name="windowTitle">
   <string>sanShip Invoice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_root">
    <property name="spacing"><number>0</number></property>
    <property name="margin"><number>0</number></property>

    <!-- Sidebar -->
    <item>
     <widget class="QFrame" name="sidebar">
      <property name="minimumWidth"><number>70</number></property>
      <property name="maximumWidth"><number>70</number></property>
      <layout class="QVBoxLayout" name="verticalLayout_sidebar">
       <property name="margin"><number>8</number></property>
       <property name="spacing"><number>8</number></property>

       <item>
        <widget class="QPushButton" name="btn_dashboard">
         <property name="text"><string>DB</string></property>
         <property name="checkable"><bool>true</bool></property>
         <property name="checked"><bool>true</bool></property>
        </widget>
       </item>

       <item>
        <widget class="QPushButton" name="btn_invoice">
         <property name="text"><string>INV</string></property>
         <property name="checkable"><bool>true</bool></property>
        </widget>
       </item>

       <item>
        <widget class="QPushButton" name="btn_debitnote">
         <property name="text"><string>DN</string></property>
         <property name="checkable"><bool>true</bool></property>
        </widget>
       </item>

       <item>
        <spacer name="verticalSpacer">
         <property name="orientation"><enum>Qt::Vertical</enum></property>
         <property name="sizeHint" stdset="0">
          <size><width>20</width><height>40</height></size>
         </property>
        </spacer>
       </item>

      </layout>
     </widget>
    </item>

    <!-- Right Area -->
    <item>
     <widget class="QFrame" name="rightFrame">
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <property name="margin"><number>10</number></property>
       <property name="spacing"><number>10</number></property>

       <!-- Top bar -->
       <item>
        <widget class="QFrame" name="topBar">
         <property name="objectName"><string>topBar</string></property>
         <layout class="QHBoxLayout" name="horizontalLayout_top">
          <property name="margin"><number>8</number></property>
          <item>
           <widget class="QLabel" name="label_companyName">
            <property name="objectName"><string>label_companyName</string></property>
            <property name="text"><string>sanShip Logistics Invoice Manager</string></property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation"><enum>Qt::Horizontal</enum></property>
            <property name="sizeHint" stdset="0">
             <size><width>40</width><height>20</height></size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>

       <!-- Pages -->
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <widget class="QWidget" name="page_dashboard"/>
         <widget class="QWidget" name="page_invoice"/>
         <widget class="QWidget" name="page_debitnote"/>
        </widget>
       </item>

      </layout>
     </widget>
    </item>

   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
